.github/workflows/windows.yml

```diff
@@ -0,0 +1,81 @@
+name: windows
+on: [push, pull_request]
+
+jobs:
+
+  test:
+    strategy:
+      matrix:
+        generator:
+          - Visual Studio 16 2019
+        configuration:
+          - Release
+    runs-on: windows-2019
+    env:
+      build_dir: .build
+    steps:
+      - name: checkout
+        uses: actions/checkout@v3
+      - name: choose Python
+        uses: actions/setup-python@v3
+        with:
+          python-version: 3.9
+      - name: learn Python cache directory
+        id: pip-cache
+        run: |
+          pip install --upgrade pip
+          echo "::set-output name=dir::$(pip cache dir)"
+      - name: restore Python cache directory
+        uses: actions/cache@v2
+        with:
+            path: ${{ steps.pip-cache.outputs.dir }}
+            key: ${{ runner.os }}-${{ hashFiles('.github/workflows/windows.yml') }}
+      - name: install Conan
+        run: pip install wheel conan
+      - name: check environment
+        run: |
+          $env:PATH -split ';'
+          python --version
+          conan --version
+          cmake --version
+          dir env:
+      - name: configure Conan
+        run: |
+          conan profile new default --detect
+          conan profile update settings.compiler.runtime=MT default
+          conan profile update settings.compiler.toolset=v141 default
+      - name: learn Conan cache directory
+        id: conan-cache
+        run: |
+          echo "::set-output name=dir::$(conan config get storage.path)"
+      - name: restore Conan cache directory
+        uses: actions/cache@v2
+        with:
+            path: ${{ steps.conan-cache.outputs.dir }}
+            key: ${{ hashFiles('~/.conan/profiles/default', 'conanfile.py', 'external/rocksdb/*', '.github/workflows/windows.yml') }}
+      - name: export RocksDB
+        run: conan export external/rocksdb
+      - name: install dependencies
+        run: |
+          mkdir $env:build_dir
+          cd $env:build_dir
+          conan install .. --build missing --settings build_type=${{ matrix.configuration }} --profile:build default --profile:host default
+      - name: configure
+        run: |
+          $env:build_dir
+          cd $env:build_dir
+          pwd
+          ls
+          cmake `
+            -G "${{ matrix.generator }}" `
+            -DCMAKE_TOOLCHAIN_FILE:FILEPATH=build/generators/conan_toolchain.cmake `
+            -Dassert=ON `
+            -Dreporting=OFF `
+            -Dunity=OFF `
+            ..
+      - name: build
+        run: |
+          cmake --build $env:build_dir --target rippled --config ${{ matrix.configuration }} --parallel $env:NUMBER_OF_PROCESSORS
+      - name: test
+        run: |
+          & "$env:build_dir\${{ matrix.configuration }}\rippled.exe" --unittest
```

Comment:
Would it be worth trying to run the jobs in parallel here, too? If nothing else, `nproc` is available in the bash shell last time I looked.

Reply:
In my testing, parallel tests were not reliable on Windows.